### ISSUE TITLE
Grad strides do not match bucket view strides

### ISSUE THREAD

**Issue by julkaztwittera** [NVlabs/stylegan2-ada-pytorch#275]
I am trying to train a model. What I get is such a warning:
`/home/jchylak/.local/lib/python3.8/site-packages/torch/autograd/__init__.py:197: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [1, 512], strides() = [1, 1]
bucket_view.sizes() = [1, 512], strides() = [512, 1] (Triggered internally at ../torch/csrc/distributed/c10d/reducer.cpp:325.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass`
Has anyone ever got it? I've seen some issues about it in PyTorch, but I don't know how to manage it here.

**Comment by rakhon9** [comment]
Dear Julia.
I am having the same error. Can you please let me know if you have solved this problem or not?
```
C:\Users\palmcove\anaconda3\envs\cuda_l\lib\site-packages\torch\autograd\__init
__.py:197: UserWarning: Grad strides do not match bucket view strides. This may
 indicate grad was not created according to the gradient layout contract, or th
at the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [1, 512], strides() = [1, 1]
bucket_view.sizes() = [1, 512], strides() = [512, 1] (Triggered internally at C
:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\torch\csrc\distributed\c10d\reducer.cpp:339.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
```
